src/reducers/entityReducers/articles.js

```diff
@@ -16,6 +16,15 @@ export default (state, action) => {
 
       return state.setIn(['article'], newArticles);
     }
+    case DELETE_ARTICLE_SUCCESS: {
+      const articleIndex = state.getIn(['article']).findIndex((el) => el.get('id') === action.meta.articleId);
+
+      if (articleIndex === MINUS_ONE) {
```

Comment:
I think it's a redundant check: you can just call `deleteIn` and it will return the same `state` if index equals -1